### PR TITLE
Correct Sunday Express bus schedule times

### LINKED ISSUE #1538: Express Bus schedule for Sunday is incorrect
For reference, this is the proper schedule linked on the website as of 2017-09-10: ![[Bus Schedule]](https://wp.stolaf.edu/sa/wp-content/blogs.dir/78/files/2012/08/Express-Bus-2016.jpg)

[data/bus-times/express.yaml](https://github.com/StoDevX/AAO-React-Native/blob/a1a9776a3847a0c46464f1431376f35497e9efe8/data/bus-times/express.yaml) is the file affected.  It appears that the times shifted forward two hours from what is currently reflected.

### PR DESCRIPTION
**Pull request by rye**
For reference, I used the [Brochure PDF](http://threeriverscap.org/sites/default/files/complete_nf_brochure.pdf) file to determine that we probably just needed to shift the hours forward two hours.  This is the change made here.  I'm unsure if I need to edit another file or if it'll get auto-generated in CI.

Closes #1538

### CODE REVIEW

**Review by hawkrives — APPROVED** [review]
Thanks!

We autogenerate the json file with the ci, yeah, so this is the only one that needs to change.

**Comment by rye** [comment]
Awesome!  Will wait for CI to pass before merging.

**Review by elijahverdoorn — APPROVED** [review]
Thanks! Good quick turn-around on the complaint we got.

**Comment by rye** [comment]
Merging now.  Thanks, all!

**Comment by hawkrives** [comment]
I've nearly got a PR ready to load the bus data from github, so we'll actually be able to update this remotely in the future…

**Pull request merged**
